paper-ieee-trans.tex

```diff
@@ -108,16 +108,16 @@ \section{Introduction}
 organized as follows. Section \ref{sec:ev} details the evolution of research on
 software reliability in EC-projects. Section \ref{sec:ntsr} provides
 information about new research trends in software reliability based upon the
-experiences from the recent {\sl INDIGO--DataCloud} project. Conclusions are drawn in Section \ref{sec:con}.
+experiences from the recent {\sl INDIGO--DataCloud} project (INDIGO--DC). Conclusions are drawn in Section \ref{sec:con}.
 
 \section{Evolution of research on software reliability in EC-projects}
 \label{sec:ev}
 
 The EC projects considered in this paper are listed chronologically in Table
 \ref{tab:eup}. Each project paved the way of the subsequent projects,
 following a logic evolution in which the role of the quality and reliability of the software have
-consistently increased. As shown in Table \ref{tab:feat}, the most recent projects are more committed and aware to software reliability practices.
-Table \ref{tab:feat} also shows the topics addressed in the following sections, highlighting
+consistently increased. As shown in Table \ref{tab:feat}, the most recent projects are more committed to and aware of software reliability practices.
+Table ~\ref{tab:feat} also shows the topics addressed in the following sections, highlighting
```

Comment:
remove tilde

Reply:
agree with change and Pablo

Reply:
committed to and aware of .... ok
I would like tilde but it depends on its use in the other ref or cite

Reply:
no tilde